analyzer/util/util.go

```diff
@@ -43,6 +43,19 @@ func NewBackoff(initialTimeout time.Duration, maximumTimeout time.Duration) (*Ba
 // Wait waits for the appropriate backoff interval.
 func (b *Backoff) Wait() {
 	time.Sleep(b.currentTimeout)
+}
+
+// Success slightly decreases the backoff interval.
+func (b *Backoff) Success() {
+	b.currentTimeout *= 9
+	b.currentTimeout /= 10
```

Comment:
Maybe parametrize the percentage to decrement?

Reply:
I think it's not worth it yet. The backoff is only used in one place, so parametrization would only really make a difference if I then exposed it all the way up through Helm. Well, it would also make for slightly cleaner code, but I don't think as-is is too bad.